project/MimaExcludes.scala

```diff
@@ -62,11 +62,66 @@ object MimaExcludes {
     ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.deploy.DeployMessages#RequestExecutors.copy"),
     ProblemFilters.exclude[IncompatibleResultTypeProblem]("org.apache.spark.deploy.DeployMessages#RequestExecutors.copy$default$2"),
     ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.deploy.DeployMessages#RequestExecutors.this"),
-    ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.deploy.DeployMessages#RequestExecutors.apply")
+    ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.deploy.DeployMessages#RequestExecutors.apply"),
+
+    // [SPARK-38679][CORE] Expose the number of partitions in a stage to TaskContext
```

Comment:
<img width="823" alt="image" src="https://user-images.githubusercontent.com/1475305/187824035-1ca93e22-1ac2-4dcb-b38f-f87115948b94.png">

It was placed in `v32excludes` before, and move to `v34excludes`  in this pr.

cc @vkorukanti and @cloud-fan SPARK-38679:

- [[SPARK-38679][CORE] Expose the number partitions in a stage to TaskContext](https://github.com/apache/spark/pull/35995#top)
#35995